cache-extension-demo/SAM/template.yaml

```diff
@@ -0,0 +1,135 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Sample AWS SAM App for Cache Extension demo application
+
+Globals:
+  Function:
+    Timeout: 10
+
+Parameters:
+  DatabaseName: 
+    Type: String
+    Description: Please enter a name for the database
+    Default: "CacheExtensions_SampleTable"
+      
+Resources:
+
+  # Cache Extension - Lambda Function
+  SampleFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: ../example-function/
+      Handler: index.handler
+      Runtime: nodejs12.x
+      Environment: 
+        Variables:
+          databaseName: 
+            Ref: DatabaseName
+      Layers: 
+        - !Ref CacheExtensionLayer
+      Policies:
+        - AWSLambdaExecute
+        - Version: '2012-10-17'
+          Statement:
+            - Effect: Allow
+              Action:
+                - dynamodb:GetItem
+                - ssm:GetParameter
+              Resource: '*'
```

Comment:
Suggest referencing the DynamoDB table/SSM resources specifically for this, otherwise the IAM policy will allow GetItem /GetParameter for all resources in the account.
You can also use SAM managed policies for this which makes it even easier: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-policy-template-list.html#dynamo-db-write-policy
https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-policy-template-list.html#ssm-parameter-read-policy